src/views/Player.vue

```diff
@@ -521,13 +524,14 @@ export default {
             }
             var modal = document.getElementsByClassName("modal")[0];
             if (modal != undefined)   {
+              this.player.pause();
+
               document
                 .getElementsByClassName("plyr")[0]
                 .appendChild(document.getElementsByClassName("modal")[0]);
 
               if (plioQuestion["state"] == "notshown") plioQuestion["state"] = "unanswered"; 
                 //var marker = plioQuestion["marker"];
-                this.player.pause();
```

Comment:
moving player.pause() upwards because sometimes the player keeps playing even when modal is visible -> but i didn't see any visible change